tests/testthat/test-derive_var_afeye.R

```diff
@@ -77,7 +77,7 @@ test_that("derive_var_afeye Test 3: Deprecation of dataset_occ", {
         loc_var = AELOC,
         lat_var = AELAT
       ),
-    class = "lifecycle_warning_deprecated"
+    class = "lifecycle_error_deprecated"
   )
 
   expect_dfs_equal(
```

Comment:
You need to delete this `expect_dfs_equal()` block of code in Test 3. This is because we have progressed the deprecation stage from "warning" to "error" and so twhile the previous behaviour of the function was to throw a warning but create the output anyway, now when you run the line of code below:
```
ctual_output3 <- expected_output3 %>%
      select(-AFEYE) %>%
      derive_var_afeye(
@@ -77,7 +77,7 @@ test_that("derive_var_afeye Test 3: Deprecation of dataset_occ", {
        loc_var = AELOC,
        lat_var = AELAT
      )
```
it errors, as expected (this is why it's wrapped in an `expect_error()`. So we can only test that an error is thrown by that line. The object `actual_output3` will not be created. Hence why you need to delete the `expect_dfs_equal()` test.